float32[] mower_gps
float32 heading_degree
float32 heading_bias
float32 obstacle_radius
string output_path